crypto-test/src/test/scala/org/bitcoins/crypto/SignTest.scala

```diff
@@ -16,7 +16,7 @@ class SignTest extends BitcoinSJvmTest {
   behavior of "Sign"
 
   it must "sign arbitrary pieces of data correctly" in {
-    forAllAsync(CryptoGenerators.sha256Digest) { hash =>
+    forAll(CryptoGenerators.sha256Digest) { hash =>
```

Comment:
All three changes made here are unsafe as nothing is being properly tested

Reply:
Can you elaborate?

Reply:
Oh yes, there are futures inside.